### PR TITLE
Hack Session 0 Page Addition

### PR DESCRIPTION
**Pull request by LOTaher**
## Description

This change is for the new git activity I will propose in today's meeting.

There are still some minor as well as some styling changes that need to be made. 

## Changes

* Added a /hs-0 route to the website

## Screenshots

image

### CODE REVIEW

File: src/app/hs-0/page.tsx [hunk 1]
```diff
@@ -38,14 +40,16 @@ const members = [
   },
 ];
 
+// The only code that should be changed below this line is on line 51.
 export default function HS0() {
   return (
     <div className="bg-[#ECF0F1]">
```
Comment (fraander) [inline src/app/hs-0/page.tsx]: There are color constants inside tailwind for all the colors. Try `bg-oasis` and then do `ctrl`+`space` to get Intellisense with all the colors autocompleted for choices.

File: src/app/hs-0/page.tsx [hunk 2]
```diff
@@ -38,14 +40,16 @@ const members = [
...
       <NavBar active="" />
       <div className="flex flex-col justify-start mt-8 mb-12">
         <h1 className="mx-auto"> 🌴 Hack Session 0 🌴</h1>
```
Comment (fraander) [inline src/app/hs-0/page.tsx]: There's an extra leading space here, heads up

File: src/app/hs-0/page.tsx (hunk 1 above)
Comment (LOTaher) [reply src/app/hs-0/page.tsx]: Fixed!

File: src/app/hs-0/page.tsx (hunk 2 above)
Comment (LOTaher) [reply src/app/hs-0/page.tsx]: Fixed!

**Pull request merged**